bup: add page (#2454)
tldr-pages/tldr: pages/common/bup.md

# bup

> > Very efficient backup system based on the git packfile format.

Reviewer: What is meant by efficient here? Efficient in terms of time, space used, or something else?

Author: Space used, it providing fast incremental saves and global deduplication.

Reviewer: Right. Perhaps this would work better then?

```suggestion
> Backup system based on the git packfile format, providing fast incremental saves and global deduplication.
```

Author: Yep this work better

- Initialize a backup repository in the specified local directory:

`bup -d {{path/to/repository}} init`

> - Index folder "my_folder" before of backup:

Reviewer: I'm not sure what this does.

Author: Index prepare the dir for the backup. Think it as git add, you need do before backup.
```suggestion
- Prepare a folder to the backup:
```

Reviewer: Ahhh, ok :-)

`bup -d {{path/to/repository}} index {{path/to/my_folder}}`

> - Backup folder "my_folder" to the repository:

Reviewer: ```suggestion
- Backup a folder to the repository:
```

Perhaps this will read better?

`bup -d {{path/to/repository}} save -n {{backup_name}} {{path/to/my_folder}}`

> - Show backup snapshots currently stored in the repository:

Reviewer: ```suggestion
- Show the backup snapshots currently stored in the repository:
```

Perhaps this will read better?
